schema.sql

```diff
@@ -9,4 +9,7 @@ CREATE TABLE animals(
     weight_kg DECIMAL
 );
 
-DROP TABLE animals;
\ No newline at end of file
+ALTER TABLE
+    animals
+ADD
+    COLUMN species varchar;
```

Comment:
- [ ] [Optional] Please remove white spaces to make your code more readable. 
You can make it as one line of code like that:
```sql
ALTER TABLE animals ADD species varchar(255);
```